### ISSUE TITLE
Stop using Libravatar

### ISSUE THREAD

**Issue by ktrvs** [decent-chat/decent#305]
Libravatar is shutting down on 2018-09-01[(blog post)](https://blog.libravatar.org/posts/Libravatar.org_is_shutting_down_on_2018-09-01/). We'll need to stop using it soon, and prepare to switch to something else as the main source of Avatars. I'm guessing it'll be Gravatar.

**Comment by towerofnix** [comment]
@ktrvs Thanks for bringing this up!

@heyitsmeuralex Thoughts on being able to directly upload avatars to the Decent server? Assuming users are allowed to upload images, anyhow. See also #101.

Since we probably won't get that done for a very, very, very long time (it's dependent on the permission system!), I recommend changing the code to fetch from Gravatar instead of Libravatar. (Note this is a server-side change - there's a function (in `server/common.js`) called `emailToAvatarURL` that should be updated.)

**Comment by towerofnix** [comment]
Of course, the relevant message in the client should be changed as well ("We use Gravatar..." instead of "We use Libravatar...").

**Comment by ktrvs** [comment]
👍🏻

**Comment by towerofnix** [comment]
@heyitsmeuralex Given I don't use *any* chat systems frequently anymore (I'm mostly lurking the main chats!), I haven't been doing much work on Decent; I've been occupied doing a lot of other stuff, which definitely includes school and sleep! :P
